kuka_kr10_support/urdf/kr10r900_2_macro.xacro

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find kuka_resources)/urdf/common_constants.xacro"/>
+  <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
```

Comment:
might want to investigate moving these two lines *inside* the macro def.

Reply:
does that make a difference?

Reply:
Yes. Newer versions of `xacro` support macro scope.